config/database.yml

```diff
@@ -1,18 +1,20 @@
 default: &default
- adapter: sqlite3
+ adapter: mysql2
+ host: localhost
+ username: root
+ password:
```

Comment:
we need to update this to use a config file, we dont want to be checking in changes to database.yml